src/main/scala/centaur/test/CheckFiles.scala

```diff
@@ -0,0 +1,82 @@
+package centaur.test
+
+import com.google.cloud.storage.{Storage, Blob, BlobId}
+import com.google.cloud.storage.Storage.BlobListOption
+import scala.collection.JavaConverters._
+import java.io.File
+
+sealed trait CheckFiles {
+  def checkDirectorySize: String => Int 
```

Comment:
With one argument it's easier to compose/andThen these if declared this way.  Otherwise you have to partially apply and it looks a little less clean.